Open Judge System/MongoDBPoC/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace MongoDBPoC
+{
+    using System;
+    using System.Collections.Generic;
+
+    using MongoDB.Bson;
+    using MongoDB.Driver;
+
+    public class Program
+    {
+        static void Main()
+        {
+            const string ConnectionString = "mongodb://localhost:27017";
+            var client = new MongoClient(ConnectionString);
+            IMongoDatabase db = client.GetDatabase("OJS");
+            var collection = db.GetCollection<SubmissionForProcessing>("submissionsForProcessing");
+
+            InsertSubmissions(collection);
+
+            var filter = new FilterDefinitionBuilder<SubmissionForProcessing>().Where(s => s.Processed);
+            collection.Find(filter).ToList()
+                .ForEach(s => Console.WriteLine(
+                             $"ID: {s.SubmissionId}; Processed: {s.Processed}; Processing: {s.Processing}"));
+
+        }
+
+        private static void InsertSubmissions(IMongoCollection<SubmissionForProcessing> collection)
+        {
+            var submissionOne = new SubmissionForProcessing()
+            {
+                Processed = false,
+                Processing = true,
+                SubmissionId = 1
+            };
+            var submissionTwo = new SubmissionForProcessing()
+            {
+                Processed = false,
+                Processing = false,
+                SubmissionId = 2
+            };
+            var submissionThree = new SubmissionForProcessing()
+            {
+                Processed = true,
+                Processing = false,
+                SubmissionId = 3
+            };
+
+            var submissions = new List<SubmissionForProcessing>()
+            {
+                submissionOne,
+                submissionTwo,
+                submissionThree
+            };
+
+            collection.InsertMany(submissions);
+        }
+
+        private class SubmissionForProcessing
+        {
```

Comment:
why not in separate class?

Reply:
No need, it will just clutter unnecessarily the project and decrease visibility. It's a miniature POCO class just for showcasing what a real simple Mongo entity looks like.